packages/client/src/StreamrClient.ts

```diff
@@ -556,12 +556,21 @@ export class StreamrClient {
 
     async setProxies(
         streamDefinition: StreamDefinition,
-        nodeIds: string[],
+        proxyNodes: JsonPeerDescriptor[],
         direction: ProxyDirection,
         connectionCount?: number
     ): Promise<void> {
         const streamPartId = await this.streamIdBuilder.toStreamPartID(streamDefinition)
-        await this.node.setProxies(streamPartId, nodeIds, direction, connectionCount)
+        await this.node.setProxies(streamPartId, proxyNodes, direction, connectionCount)
+    }
+
+    /**
+     * Used to set known entry points for a stream partition. If entry points are not set they 
+     * will be automatically discovered from the Streamr Network.
+    */
+    async setStreamPartEntryPoints(streamDefinition: StreamDefinition, entryPoints: JsonPeerDescriptor[]): Promise<void> {
```

Comment:
Does an entry point exists so that peers can find other nodes which are subscribers/publishers of a stream? If that is the case, the entry points are stream-specific, not streamPart-specific?

The network has always operated on streamPart-level, but the client uses higher abstraction level. Therefore the API which we provide to users should maybe use streams.